spec/features/consumer/shopping/embedded_shopfronts_spec.rb

```diff
@@ -110,6 +105,16 @@
     end
   end
 
+  private
+  
+  def on_embedded_page
+      expect(page).to have_selector "iframe"
+
+      within_frame :frame do
+        yield
+      end
+  end
```

Comment:
I'd DRY it by moving its definition to `spec/support/`

Reply:
Now that I see, this might only be used in this file. Then 👍

Reply:
This was used in two files, so I moved it to a helper to take the refactoring another step further. I also removed the expectation which I think isn't needed. I'm not sure why @Matt-Yorkley did it in https://github.com/mkllnk/openfoodnetwork/commit/113f6565be71cf2b2cf2ca0c3ee622f576f8e1fc#diff-2f114975ce4c238323ecb0c23daab4b7R47, but it could be, because he used the web driver directly to load a local file and wanted to assert that it worked.